### DISCUSSION TITLE
keyball44 キーが反応しない & 基盤はがれ

### DISCUSSION THREAD

**Discussion by r3ucht2v** [Yowkees/keyball#488] [Q&A] [answered]
Keyball44について、質問があります。

青枠で囲ったキーが反応しません(デフォルトでCのキー)。キーソケットを差し替えましたが、変わらず反応しません。

また、キーソケットを取り替えるときに赤枠の金属メッキが剥がれてしまいました。
こちらのキーを復活させるために、繋ぐ場所があれば教えていただきたいです。

対応するダイオードなどやチェックする項目があれば、教えていただけると幸いです。

よろしくお願いいたします。

![IMG_20240222_170301398 (1)](https://github.com/Yowkees/keyball/assets/95508383/8f5c2e39-7fe2-4c9b-a2ad-ca1727b0f9ed)
![IMG_20240222_170704755 (1)](https://github.com/Yowkees/keyball/assets/95508383/6b8be2be-5a50-48d6-bba7-1bb372ba5619)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

添付しました写真の赤点どうしをエナメル線か、ゴム被覆線でつないでみてください。

![306917153-8f5c2e39-7fe2-4c9b-a2ad-ca1727b0f9ed (1)](https://github.com/Yowkees/keyball/assets/143698349/a13fb430-b7e3-4a6b-8bbe-452a63d2df50)

**Reply by r3ucht2v** [reply]
ご回答ありがとうございます。
早速ためしてみましたが、Cを入力すると、Cは入力されることなく、今回示してもらった赤点相方のDが入力される結果となりました。
他に考えられる方法はありますでしょうか。

よろしくお願いいたします。

**Reply by Ryo624** [reply, marked as answer]
申し訳ございません、勘違いしておりました。

こちらの写真の赤点どうしをつないでみるとどうでしょうか。
![306917153-8f5c2e39-7fe2-4c9b-a2ad-ca1727b0f9ed](https://github.com/Yowkees/keyball/assets/143698349/423aef69-3dc1-427f-a5cc-053e292fcd33)

**Reply by r3ucht2v** [reply]
問題は解決しました。

ありがとうございました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
ぜひ末永くご愛用いただければ幸いです。
